mantidproject/mantid
# Added MSD specific files

**Description of work.**

This updates the MSD fit tab to use the simultaneous fitting options available in the other tab in the Indirect Data Analysis GUI. 



**To test:**
  * Code review
  * I have a test file from a scientist but am unsure if it is shareable to slack me and I'll send it to you.
  * Load file
  * Changed range to Fit x-range is 0 to 0.85.
  * Start with function MsdGauss – parameters height=0.05 msd=3.09. Should give a good fit.
  * Change MsdYi. It will have the 2 fitted values as above plus sigma=1.0. Fit should give sigma=4.75
  * Go to simultaneous & make sigma global.
  * Try some other sequences of actions. Particularly ones you think might break things.



Fixes #26880 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
